src/v2/util/isMobile.ts

```diff
@@ -0,0 +1,6 @@
+export function isMobile() {
+  return (
+    typeof window.orientation !== 'undefined' ||
+    navigator.userAgent.indexOf('IEMobile') !== -1
```

Comment:
Shouldn't you be detecting touch?

Reply:
Apparently the presence of  `window.orientation` gets the most coverage for mobile devices. the `IEMobile` is just to catch that Windows mobile devices.

Reply:
I don't get it though — this is explicitly because of touch devices. It has nothing to do with mobility.

Reply:
What about touch desktops?

Reply:
Ok, good point